docs/content/en/docs/reference/changelog.md

```diff
@@ -6,6 +6,32 @@ weight: 35
 
 ## Unreleased
 
+## [v0.15.0](https://github.com/aws/eks-anywhere/releases/tag/v0.15.0)
+### Added
+- Kubernetes 1.26 support ([#5079](https://github.com/aws/eks-anywhere/issues/5079))
+- Workload clusters full lifecycle API support for Bare Metal provider ([#5237](https://github.com/aws/eks-anywhere/issues/5237))
+- Workload clusters full lifecycle API support for Nutanix provider ([#5190](https://github.com/aws/eks-anywhere/pull/5190))
+- Support for InsecureSkipVerify for RegistryMirrorConfiguration across all providers. Currently not support for Bottlerocket OS ([#1647](https://github.com/aws/eks-anywhere/issues/1647))
+- Support for configuring of Bottlerocket settings. ([#707](https://github.com/aws/eks-anywhere/issues/707))
+- Support for using a custom CNI ([#5217](https://github.com/aws/eks-anywhere/issues/5217))
+- Ability to configure NTP servers on EKS-A nodes for vSphere and Tinkerbell providers ([#4760](https://github.com/aws/eks-anywhere/issues/4760))
```

Comment:
```suggestion
- Ability to configure NTP servers on EKS Anywhere nodes for vSphere and Tinkerbell providers ([#4760](https://github.com/aws/eks-anywhere/issues/4760))
```
We try to spell out "EKS Anywhere" in general, unless of course it is the literal name of a path or artifact name.